Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/menu/ui/TestMenuFragmentViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package de.rki.coronawarnapp.test.menu.ui
+
+import androidx.lifecycle.MutableLiveData
+import com.squareup.inject.assisted.AssistedInject
+import de.rki.coronawarnapp.test.api.ui.TestForAPIFragment
+import de.rki.coronawarnapp.test.risklevel.ui.TestRiskLevelCalculationFragment
+import de.rki.coronawarnapp.util.ui.SingleLiveEvent
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModel
+import de.rki.coronawarnapp.util.viewmodel.SimpleCWAViewModelFactory
+
+class TestMenuFragmentViewModel @AssistedInject constructor() : CWAViewModel() {
+
+    val testMenuData = MutableLiveData<List<TestMenuItem>>()
+    val showTestScreenEvent = SingleLiveEvent<TestMenuItem>()
+
+    init {
+        loadMenu()
```

Comment:
please not in constructor - maybe create load method to call from <fragment>.onViewCreated()?